packages/styles/variables.css

```diff
@@ -50,6 +50,12 @@
   --header-text-color: var(--gray-80);
   --header-text-color-dark: var(--gray-90);
 
+  /* stroke colours */
+  --stroke-light: #b3bfc6;
+
+  /* background colours */
+  --background-light: #f0f2f5;
```

Comment:
@bobbyomari Just wondering why one of the existing "gray"'s isn't sufficient?

Reply:
We have had these colors in our designs that appear to have never been translated into variables (and use them quite frequently throughout light theme components/styles). Aaron and I developed these colors `stroke-dark` and `stroke-light` because the colors for both light and dark themes vary in contrast. It is not a 1:1 match between both themes.

Reply:
@bobbyomari Sorry, was specifically trying to ask about the `--background-light` color. Typically, our backgrounds are either white or one of the grays

Reply:
> @bobbyomari Sorry, was specifically trying to ask about the --background-light color. Typically, our backgrounds are either white or one of the grays

That is not entirely true - there are layouts and designs that do use `--background-light`. It's likely it just was never implemented correctly and/or the color variable was never implemented. In fact, our navbar, topbar, and footer components are not using white `#ffffff` and using `#fdfdfe` for light theme. It's subtle, but there are use cases where `--background-light` is being used in our applications such as codeblocks and various background colors that only apply to light theme styles. 

I do have a color palette included on that documentation I previously included, but I think it will help to separate them into Dark and Light theme categories for clarity and examples of where they are used.

Reply:
Thanks for the explanation.